test/test_main.py

```diff
@@ -17,8 +17,16 @@
 except NameError:
     text_type = str
 
+test_dir = os.path.abspath(os.path.dirname(__file__))
+project_dir = os.path.dirname(test_dir)
+default_config_file = os.path.join(project_dir, 'mycli', 'myclirc')
+login_path_file = os.path.join(test_dir, 'mylogin.cnf')
+
+os.environ['MYSQL_TEST_LOGIN_FILE'] = login_path_file
 CLI_ARGS = ['--user', USER, '--host', HOST, '--port', PORT,
-            '--password', PASSWORD, '_test_db']
+            '--password', PASSWORD, '--myclirc', default_config_file,
+            '--defaults-file', default_config_file,
```

Comment:
I have the unit tests using the default `myclirc` as the mysql `my.cnf` because we need some kind of cnf-type file to override the user's file, but we don't actually need to set any config in it. Since `myclirc` is the same format as `my.cnf` and because it doesn't use the `[mysql]` group, I figured we could use it rather than adding another file to the repo.